### ISSUE TITLE
Authorization type

### ISSUE THREAD

**Issue by stefankusmierz** [rdmorganiser/rdmo#616]
Hi,
is RDMO's api compatible with an authorization using an identity provider via saml2?

**Comment by jochenklar** [comment]
I don't think so, we can do Shibboleth as documented here: https://rdmo.readthedocs.io/en/latest/configuration/authentication/shibboleth.html.

@MyPyDavid has a setup running with Keycloak, maybe that helps.

**Comment by MyPyDavid** [comment]
Hi, 
I think that the current Shibboleth implementation (from https://rdmo.readthedocs.io/en/latest/configuration/authentication/shibboleth.html) intercepts the requests, to `api/v1/projects/...`  for example, because of:
```bash
<LocationMatch /(account|domain|options|projects|questions|tasks|conditions|views)>
```
The `/api/v1/core/settings/` should be still reachable...

Another implementation of shibboleth could also solve this issue @jochenklar

**Comment by stefankusmierz** [comment]
@MyPyDavid : Hi, which protocol did you use for keycloak? saml or OpenID Connect?

**Comment by MyPyDavid** [comment]
I have added keycloak with OpenID Connect to RDMO. In the last Community Meeting I did a presentation about this: https://www.forschungsdaten.org/index.php/Datei:2023-03-29_RDMO-Community-Treffen_Local-AAI-Keycloak_DavidWallace.pdf

**Comment by MyPyDavid** [comment]
this may soon also be helpful for this issue: https://github.com/pennersr/django-allauth/pull/3340

**Comment by stefankusmierz** [comment]
Thanks for the input, David!

**Comment by stefankusmierz** [comment]
Thanks again, David!
